### ISSUE TITLE
Unable to input Nordic characters

### ISSUE THREAD

**Issue by axolul** [magmax/python-inquirer#480]
**Enviroment:**

- Windows 10 Pro 22H2
- Python 3.11.6
- inquirer 3.1.3

**Code:**

```py
import inquirer

questions = [
    inquirer.Text('newUUID', message="Anna uusi UUID (paina Enter pitääksesi samana): "),
    inquirer.Text('newName', message="Anna uusi nimi (paina Enter pitääksesi samana): "),
    inquirer.Text('newPrice', message="Anna uusi hinta (paina Enter pitääksesi samana): "),
    inquirer.Text('newCategory', message="Anna uusi kategoria (paina Enter pitääksesi samana): "),
]
```

We are using inquirer in our school project. It shows Finnish/Nordic characters ä, ö and å normally when it prompts for something. But then the command line goes nuts when you try to input the same characters.

In the picture attached, I inputted "testöäåtest". As you can see, the inquiry line starts to multiply and it goes all wrong from there.

![nordic_characters](https://github.com/magmax/python-inquirer/assets/130797534/734b3995-6a90-4eb7-807e-5e3c0d43646c)

**Comment by Cube707** [comment]
Duplicate of #432

This is a upstream issue with the `readchar` library, see https://github.com/magmax/python-readchar/issues/93

**Comment by Cube707** [comment]
Duplicate of #432

**Comment by Cube707** [comment]
closed by #622
